src/appHost.tsx

```diff
@@ -944,13 +954,27 @@ miss: ${memoizedWithMissHit.miss}
                 observable.subscribe(shell, () => {
                     flushMemoizedForState()
                 })
```

Comment:
A note for the some time in the future - 
We'd probably want to make sure `flushMemoizedForState` is not called in vein, (e.g. 2 observers where updates, and we delay the subscription trigger to animation frame anyway - w'd call the flush twice in a row)